docs/DeveloperGuide.md

```diff
@@ -154,6 +154,28 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Edit Feature
+The `edit` feature allows users to update specific information about certain employees. This feature is supported by 
+three classes, `EditCommand`, `EditCommandParser` and `Model`.
+
+#### Implementation
+
+`EditCommand`: Stores the edited changes to be changed.<br>
+`EditCommandParser`: Parses the user edit command to create an appropriate EditCommand object.<br>
```

Comment:
Should "the user edit command" be "the user's edit command"

Reply:
Ok updated!